lib/Aftok/Orphans.hs

```diff
@@ -0,0 +1,8 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Aftok.Orphans where
```

Comment:
What is Orphans for? Could we maybe put a short blurb at the top of new modules to explain purpose?